src/scripts/main.js

```diff
@@ -1,5 +1,25 @@
 'use strict';
 
-// const pushNotification = (top, right, title, description, type) => {
-//   // write code here
-// };
+const pushNotification = (top, right, title, description, type) => {
+  const body = document.querySelector('body');
```

Comment:
Тут одразу можна звертатися так як на тому ж рівні
```suggestion
  const body = document.body;
```
Тоді вже не обов'язково змінну створювати..

Reply:
fixed